packages/utilities/src/AutoScroll.ts

```diff
@@ -31,6 +31,7 @@ export class AutoScroll {
     this._incrementScroll = this._incrementScroll.bind(this);
     this._scrollRect = getRect(this._scrollableParent);
 
+    // tslint:disable-next-line:no-any
     if (this._scrollableParent === window as any) {
```

Comment:
Why do we have to cast to `any`? Can't we update the typing of `_scrollableParent`?

Reply:
To include `Window`? Line 97 accesses property `scrollTop`, which is not a property of `Window`. Let me know if there's something else I should try

 https://github.com/erichdev/office-ui-fabric-react/blob/352a010572004287d4e12813293653dc58172ad9/packages/utilities/src/AutoScroll.ts#L97